src/envoy/mixer/tcp_filter.cc

```diff
@@ -159,6 +169,9 @@ class TcpInstance : public Network::Filter,
       if (!calling_check_) {
         filter_callbacks_->continueReading();
       }
+      report_timer_ = mixer_control_.options().env.timer_create_func(
```

Comment:
Since this inside Envoy code, it is prefer to use Envoy timer instead of mixerclient timer.